### ISSUE TITLE
Book 05: Sentence 021

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#22]
![tree](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/2e8ea56b-20f3-469d-8a42-386c3b8acf35)

```
# sent_id = Book_05_-_The_Omens.txt:21
# text = Auh intla cana oimpan ioac, cana quaujtl itzintlan, anoço atlauhcamac, ommololoa, onmocemololoa, ommotepeuhtitlalia, monechicoa, qujcujtlalpia, qujcencujtlalpia, qujcemjlpia, qujcemmana in jntopil, in jpan qujxeoaia in inteouh: Yiacatecutli.
# text[norm] = Auh intla cana o impan yoac, cana cuauhuitl itzintlan, anozo atlauhcamac, ommololoa, onmocemololoa, ommotepeuhtitlalia, monechicoa, quicuitlalpiya, quicencuitlalpiya, quicemilpiya, quicemmana in intopil, in ipan quixehuaya in inteouh: Yiacateuctli.
# text[orig] = Auh·intlacana·oimpan·ioac·,·cana·quaujtl·itzintlan·,·anoço·a¶tlauhcamac·,·ommololoa·,·onmoce¶mololoa·,·ommotepeuhtitlalia·,·mo¶nechicoa·,·qujcujtlalpia·,·qujcen¶cujtlalpia·,·qujcemjlpia·,·qujcem¶mana·injntopil·,·injpanqujxeoaia¶ininteouh·:·Yiacatecutli·.
# text[eng] = And if somewhere on them in the night, somewhere under a tree, or at the mouth of a gorge, they [...] their staffs, on which they represented their god: Yiacateuctli.
# text[osp] = Y donde qujera que llegauan a dormir aquel dia: ora fuese de baxo de algun arbol, o debaxos de alguna lapa, o en alguna cueua: luego iuntauan todos sus bordones, o cañas de camjno que llenauan y los atauan todos iuntos, en vna gaujlla: entonce dezian, que aquellos topiles, asi todos atados iuntos: eran la ymagen de su dios Yiacatecutli: (que es el de los mercaderes, y tratantes).
# references = A&D/163:5
1	Auh	auh	CCONJ	_	_	15	cc	_	Folio=3v|Paragraph=2|Line=50|Norm=Auh|Gloss=y
2	intla	tla	PRON	_	_	6	mark	_	Orig=intlacana|Folio=3v|Paragraph=2|Line=50|Norm=intla|Gloss=algo
3	cana	cana	ADV	_	_	6	advmod	_	Orig=intlacana|Folio=3v|Paragraph=2|Line=50|Norm=cana|Gloss=alguna.parte
4-5	oimpan	_	_	_	_	_	_	_	_
4	o	o		_		6	aux	_	Orig=intlacana|Folio=3v|Paragraph=2|Line=50|Norm=o|Analysed=Yes
5	impan	pan	NOUN	_	NounType=Relat|Number[psor]=Plur|Person[psor]=3|State=Pos	6	obl	_	Orig=intlacana|Folio=3v|Paragraph=2|Line=50|Norm=impan|Analysed=Yes
6	ioac	yohuac|yohua	VERB	_	_	15	advcl	_	Folio=3v|Paragraph=2|Line=50|Norm=yoac
7	,	,	PUNCT	_		15	punct	_	Folio=3v|Paragraph=2|Line=50|Norm=,|Analysed=Yes
8	cana	cana	ADV	_	_	15	advmod	_	Folio=3v|Paragraph=2|Line=51|Norm=cana|Gloss=alguna.parte
9	quaujtl	cuauhuitl	NOUN	_	State=Abs	10	nmod	_	Folio=3v|Paragraph=2|Line=51|Norm=cuauhuitl|Analysed=Yes
10	itzintlan	tzintlan	NOUN	_	NounType=Relat	15	obl	_	Folio=3v|Paragraph=2|Line=51|Norm=itzintlan|Locative=Yes
11	,	,	PUNCT	_		13	punct	_	Folio=3v|Paragraph=2|Line=51|Norm=,|Analysed=Yes
12	anoço	anozo	ADV	_		13	cc	_	Folio=3v|Paragraph=2|Line=51|Norm=anozo|Analysed=Yes
13	atlauhcamac	atlauhcamac	NOUN	_	NounClass=Loc	10	conj	_	Orig=a¶·tlauhcamac|Folio=3v,3v|Paragraph=2,2|Line=51,52|Norm=atlauhcamac|Locative=Yes|Root=atlauh|Analysed=Yes
14	,	,	PUNCT	_		15	punct	_	Folio=3v|Paragraph=2|Line=52|Norm=,|Analysed=Yes
15	ommololoa	ololoa	VERB	_	_	0	root	_	Folio=3v|Paragraph=2|Line=52|Norm=ommololoa
16	,	,	PUNCT	_		17	punct	_	Folio=3v|Paragraph=2|Line=52|Norm=,|Analysed=Yes
17	onmocemololoa	ololoa	VERB	_	Movement=And|Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	15	conj	_	Orig=onmoce¶·mololoa|Folio=3v,3v|Paragraph=2,2|Line=52,53|Norm=onmocemololoa|Incorp[adv]=cen|Analysed=Yes
17.1	cem	cen	ADV	_	_	_	_	_	Incorporated=Yes
18	,	,	PUNCT	_		19	punct	_	Folio=3v|Paragraph=2|Line=53|Norm=,|Analysed=Yes
19	ommotepeuhtitlalia	tepeuhtitlalia	VERB	_	Movement=And|Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	15	conj	_	Folio=3v|Paragraph=2|Line=53|Norm=ommotepeuhtitlalia|Analysed=Yes
20	,	,	PUNCT	_		21	punct	_	Folio=3v|Paragraph=2|Line=53|Norm=,|Analysed=Yes
21	monechicoa	nechicoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	15	conj	_	Orig=mo¶·nechicoa|Folio=3v,3v|Paragraph=2,2|Line=53,54|Norm=monechicoa|Gloss=se.junta|Analysed=Yes
22	,	,	PUNCT	_		23	punct	_	Folio=3v|Paragraph=2|Line=54|Norm=,|Analysed=Yes
23	qujcujtlalpia	cuitlalpiya	VERB	_	Number[iobj]=Sing|Number[subj]=Plur,Sing|Person[iobj]=3|Person[subj]=3|Subcat=Ditr|Tense=Pres|VerbForm=Fin	15	conj	_	Folio=3v|Paragraph=2|Line=54|Norm=quicuitlalpiya|Root=piya|Analysed=Yes
24	,	,	PUNCT	_		25	punct	_	Folio=3v|Paragraph=2|Line=54|Norm=,|Analysed=Yes
25	qujcencujtlalpia	cuitlalpiya	VERB	_	Number[iobj]=Sing|Number[subj]=Plur,Sing|Person[iobj]=3|Person[subj]=3|Subcat=Ditr|Tense=Pres|VerbForm=Fin	23	conj	_	Orig=qujcen¶·cujtlalpia|Folio=3v,3v|Paragraph=2,2|Line=54,55|Norm=quicencuitlalpiya|Incorp[advmod]=cen|Root=piya|Analysed=Yes
25.1	cen	cen	ADV	_	_	_	_	_	Incorporated=Yes
26	,	,	PUNCT	_		27	punct	_	Folio=3v|Paragraph=2|Line=55|Norm=,|Analysed=Yes
27	qujcemjlpia	ilpia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	23	conj	_	Folio=3v|Paragraph=2|Line=55|Norm=quicemilpiya|Incorp[advmod]=cen|Root=ilpiya|Analysed=Yes
27.1	cem	cen	ADV	_	_	_	_	_	Incorporated=Yes
28	,	,	PUNCT	_		29	punct	_	Folio=3v|Paragraph=2|Line=55|Norm=,|Analysed=Yes
29	qujcemmana	cemmana	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	23	conj	_	Orig=qujcem¶·mana|Folio=3v,3v|Paragraph=2,2|Line=55,56|Norm=quicemmana|Incorp[advmod]=cen|Root=mana|Analysed=Yes
30	in	in	DET	_	_	31	det	_	Orig=injntopil|Folio=3v|Paragraph=2|Line=56|Norm=in
31	jntopil	topilli	NOUN	_	Number[psor]=Plur|Person[psor]=3|State=Pos	23	conj	_	Orig=injntopil|Folio=3v|Paragraph=2|Line=56|Norm=intopil|Analysed=Yes
32	,	,	PUNCT	_		35	punct	_	Folio=3v|Paragraph=2|Line=56|Norm=,|Analysed=Yes
33	in	in	SCONJ	_	_	35	mark	_	Orig=injpanqujxeoaia¶|Folio=3v|Paragraph=2|Line=56|Norm=in
34	jpan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	35	obl	_	Orig=injpanqujxeoaia¶|Folio=3v|Paragraph=2|Line=56|Norm=ipan|Analysed=Yes
35	qujxeoaia	ixehua	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	31	acl	_	Orig=injpanqujxeoaia¶|Folio=3v|Paragraph=2|Line=56|Norm=quixehuaya|Analysed=Yes
36	in	in	DET	_	_	37	det	_	Orig=ininteouh|Folio=4r|Paragraph=1|Line=1|Norm=in
37	inteouh	teotl	NOUN	_	Number[psor]=Plur|Person[psor]=3|State=Pos	35	obj	_	Orig=ininteouh|Folio=4r|Paragraph=1|Line=1|Norm=inteouh|Analysed=Yes
38	:	:	PUNCT	_		39	punct	_	Folio=4r|Paragraph=1|Line=1|Norm=:|Analysed=Yes
39	Yiacatecutli	Yacatecuhtli	PROPN	_		37	appos	_	Folio=4r|Paragraph=1|Line=1|Norm=Yiacateuctli|Analysed=Yes
40	.	.	PUNCT	_		15	punct	_	Folio=4r|Paragraph=1|Line=1|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
López Austin has:
> Y si en algún lúgar les anochecía, al pie de algún árbol o en una cueva se juntaban; ahí se juntaban todos; ahí se reunían, se reco­gían. Ataban por los cabos, ataban juntos por los cabos, ataban juntos, asían juntos sus bastones que representaban a su dios, Yacateuctli

**Comment by ftyers** [comment]
Notes:
- Like with *nican Nueva España*, *cana cuauhuitl itzintlan* should be `obl(cana, itzintlan)`, cf. #10 
- Should *anozo* be `CONJ`? Andrews (2003) has it as a contraction of *ahno zo* (p.43) but I can't make sense of his description on pp.549-550
- Those `conj` should probably be `parataxis:shared`
- Of which verb is *in intopil* the object, if any?
- Structure of *in ipan quixehuaya in inteouh: Yiacateuctli* looks good.

**Comment by Lguyogiro** [comment]
> Of which verb is in intopil the object, if any?

quicemana

**Comment by ftyers** [comment]
> > Of which verb is in intopil the object, if any?
> 
> quicemana
> 

What are the objects of the other three verbs? *quicuitlalpiya, quicencuitlalpiya, quicemilpiya[, quicemmana in intopil]* 

Is not *in intopil* the object of all of those?

**Comment by ftyers** [comment]
How about this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/d8153642-9d89-4bef-96c2-10b74ff145e6)

**Comment by Lguyogiro** [comment]
Yeah this LGTM

**Comment by ftyers** [comment]
Merged in 576aaa5.
